Daytime-Nighttime-Interactions/fault-reporting.rst

```diff
@@ -24,118 +24,105 @@
 
-- This is an example bullet of a prerequisite (Telescope azimuth must be at 0 degrees.)
-
-  - This is an example describing an exception (If the dome is closed, telescope elevation position is not important.)
-
-- This is another example bullet of a prerequisite (Notification to specified SLACK channel.)
-- This example refers to a required action at the beginning of the procedure (It is critical the correct status of the equipment. This will be verified during :ref:`a critical step <Title-of-Procedure-Critical-Step-1>` in the procedure.)
-
-.. _Title-of-Procedure-Post-Conditions:
-
-Post-Condition
-^^^^^^^^^^^^^^
-
-.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
-.. It is preferred to include them as a bulleted or enumerated list.
-.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
-
-- This is an example bullet of a post-condition (Telescope azimuth is 0 degrees.)
-- This is another example of a post-condition (This procedure leaves the telescope with the E-stop activated.)
-
-.. _Title-of-Procedure-Procedure-Steps:
+Upon navigating to the `(OBS) <https://jira.lsstcorp.org/projects/OBS/issues/OBS-4?filter=allopenissues>`__ project in JIRA, click the "create" option on the right side of the top tool-bar.
 
-Procedure Steps
-^^^^^^^^^^^^^^^
+.. figure:: ./_static/Fault_report_example_page_1.png
+    :name: Fault-report-example-page-1
+    :target: http://target.link/url
 
-.. todo::
-   Include utilization of admonishments (caution, warning, etc.)
+    Screenshot of an example fault report. 
+- Project field: The reporter should ensure that the OBS project is selected to include all things affecting nighttime operations.
 
-.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
-.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
-.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+- Issue type: A "problem" issue type usually refers to a hardware issue. A "bug" issue type typically refers to a software issue. 
```

Comment:
Suggestion:

```
Issue type: "Problem" usually refers to a hardware issues whereas "Bug" typically refers to a software issue.
```

To you want to mention the other two types available? If so, you may want to itemize it, e.g.;

```
- Issue type:
  - "Problem", usually refers to hardware issues.
  - "Bug", typically refers to software issues.
  - "Improvement", suggestions for improvements to a procedure, software or else.
  - "Information", ???
```